src/ray/object_manager/object_manager.cc

```diff
@@ -387,6 +404,18 @@ void ObjectManager::Push(const ObjectID &object_id, const ClientID &client_id) {
     return;
   }
 
+  // Keep track of the fact that we're attempting to push this object to the
+  // remote object manager. Note that it's possible that the transfer will fail,
+  // but the object ID will eventually be cleared out of this list by the
+  // recent_pushes_timer_ timer.
```

Comment:
Pushes failing even indefinitely is fine the node will eventually trigger a pull right?

Reply:
Yeah, the recipient who needs the object should continually retry.